### PR TITLE
include test playbooks dependencies fix

### PR DESCRIPTION
**Pull request by ilaner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5194?filter=-1&jql=reporter%20%3D%20esharf

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Tests/scripts/create_artifacts_graph/create_artifacts.py [hunk 1]
```diff
@@ -6,25 +6,33 @@
...
-from demisto_sdk.commands.common.tools import get_content_path
+from demisto_sdk.commands.common.tools import get_content_path, str2bool
 
 import json
 
 logging_setup(3)
 install_logging("create_artifacts.log", logger=logger)
 
 
+def str2boolean(v: str):
```
Comment (yaakovpraisler) [inline Tests/scripts/create_artifacts_graph/create_artifacts.py:17]: Please remove if you're using the `str2bool` from tools

File: Tests/scripts/create_artifacts_graph/create_artifacts.py [hunk 2]
```diff
@@ -51,6 +60,9 @@ def main():
     parser.add_argument("-mp", "--marketplace", type=MarketplaceVersions, help="marketplace version", default="xsoar")
     parser.add_argument("-ao", "--artifacts-output", help="Artifacts output directory", required=True)
     parser.add_argument("-do", "--dependencies-output", help="Dependencies output directory", required=True)
+    parser.add_argument(
+        "-bu", "--bucket-upload", help="Upload to bucket", type=lambda x: str2bool(x or False), default=False
```
Comment (yaakovpraisler) [inline Tests/scripts/create_artifacts_graph/create_artifacts.py:64]:
```suggestion
        "-bu", "--bucket-upload", help="Upload to bucket", type=str2bool, default=False
```
That should work too

Comment (ilaner) [reply Tests/scripts/create_artifacts_graph/create_artifacts.py:64]: The bucket upload value is empty string, so it fails.

**Review by yaakovpraisler — APPROVED** [review]

**Review by daryakoval — APPROVED** [review]

**Comment by ilaner** [comment]
Forced merged because of bug with check if the build is finished

**Pull request merged**